symphony/content/content.blueprintsdatasources.php

```diff
@@ -168,7 +168,7 @@ public function __form()
 
                             // Handle pre Symphony 2.2.1 Static DS's
                         } else {
-                            $fields['static_xml'] = trim($existing->grab());
+                            $fields['static_xml'] = trim($existing->execute());
```

Comment:
Oh you're right. BTW @brendo I am working with @ChoboHub . Those issues where raised by scrutinizer, which does not always get it right. Sorry for have wasted your time on this :(